### PR TITLE
three way to insert css (stylesheet)

### PR DESCRIPTION
**Pull request by iamgique**

### CODE REVIEW

File: content/frontend-web/insert-css.md [hunk 1]
```diff
@@ -0,0 +1,79 @@
+---
+title: Three way to insert css
+showMetadata: true
+editable: true
+showToc: true
+order: 1
+---
+
+# Three ways to insert CSS (Stylesheet)
+When browser read style sheet it will manage format the HTML doc to the information is style sheet.
```
Comment (mildronize) [inline content/frontend-web/insert-css.md]: Incorrect English Grammar, most context is confuse, you can use Thai in .NET Thailand

File: content/frontend-web/insert-css.md [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+### yourstyle.css
+```cs
+h1 {
+   font-size: 18px;
+   color: #ffffff;
+}
+p {
+   padding: 5px 5px 5px 5px;
+}
+```
```
Comment (mildronize) [inline content/frontend-web/insert-css.md]:
```
->> use  ```css  , not  ```cs
```

File: content/frontend-web/insert-css.md [hunk 3]
```diff
@@ -0,0 +1,79 @@
...
+<html>
+<head>
+   <link rel="stylesheet" type="text/css" href="yourstyle.css">
+</head>
+<body>
+   <h1>External CSS</h1>
+   <p>iamgique</p>
+</body>
+</html>
+```
```
Comment (mildronize) [inline content/frontend-web/insert-css.md]:
```
->> use  ```css  , not  ```cs
```

File: content/frontend-web/insert-css.md [hunk 4]
```diff
@@ -0,0 +1,79 @@
...
+      padding: 5px 5px 5px 5px;
+   }
+</style>
+</head>
+<body>
+   <h1>External CSS</h1>
+   <p>iamgique</p>
+</body>
+</html>
+```
```
Comment (mildronize) [inline content/frontend-web/insert-css.md]:
```
->> use  ```css  , not  ```cs
```

File: content/frontend-web/insert-css.md [hunk 5]
```diff
@@ -0,0 +1,79 @@
...
+<!DOCTYPE html>
+<html>
+<head>
+</head>
+<body>
+   <h1 style="font-size: 18px; color: #ffffff;">External CSS</h1>
+   <p style="padding: 5px 5px 5px 5px;">iamgique</p>
+</body>
+</html>
+```
```
Comment (mildronize) [inline content/frontend-web/insert-css.md]:
```
->> use  ```css  , not  ```cs
```

**Review by mildronize — CHANGES_REQUESTED** [review]
Thank you for your PR.

I have some suggestion that you should fix your content

The topic should be changed, I think "Importing CSS to HTML"

The rest is ok, it would be better if you give some brief explanation on each context. Thank you very much.

Happy Hacktoberfest

**Comment by iamgique** [comment]
@mildronize Thank you for your suggestion, I've done.

**Review by mildronize — APPROVED** [review]

**Comment by mildronize** [comment]
Good Job. Thank you for your contribution. Merged. @iamgique

**Pull request merged**